OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -186,8 +186,11 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 								server.LobbyInfo.Clients.Remove(occupant);
 								server.SyncLobbyClients();
 								var ping = server.LobbyInfo.PingFromClient(occupant);
-								server.LobbyInfo.ClientPings.Remove(ping);
-								server.SyncClientPing();
+								if (ping != null)
+								{
+									server.LobbyInfo.ClientPings.Remove(ping);
```

Comment:
This was not detected by Coverity. It is based on my investigation of a reported stacktrace.